Src/BootCamp.Chapter/PeopleAndBalanceManager.cs

```diff
@@ -0,0 +1,114 @@
+﻿namespace BootCamp.Chapter
+{
+    public class PeopleAndBalanceManager
+    {
+        private readonly Person[] _peopleAndBalances;
+        public PeopleAndBalanceManager(string[] peopleAndBalances)
+        {
+            _peopleAndBalances = ArrayHandler.CreatePersonsArray(peopleAndBalances);
+        }
+
+        public string GetHighestBalanceEverAnswer()
+        {
+            return AnswerFormatter.GetFormattedAnswerForHighestBalanceEver(_peopleAndBalances, FindHighestBalanceEver());
+        }
+
+        public string GetPersonWithBiggestLossAnswer()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetPersonWithBiggestLossAnswer' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5070129)